### ISSUE TITLE
FIRST ISSUE - STRUCTURE OF THE APP

### ISSUE THREAD

**Issue by hell0fri3nd** [aaandri98/FablabNetworkMobile#10]
### STRUCTURAL QUESTION
How do we implement the app?
Two different navbars depending on the login (if fablab or user)?

**Comment by aaandri98** [comment]
Yes, I'd develop two different applications, with features in common. The fablab part of the app will be bigger and with more functionalities, seen that its admin should be able to do a lot of things.

Then, depending on the login made, you can lock/unlock functions.

**Comment by hell0fri3nd** [comment]
We will have two different navbars, etc.. 2 apps in one, challenging!

**Comment by aaandri98** [comment]
i think it's the only way to do it.. do you have other ideas?

we can also have a bigger application, but with some function disabled in one or the other version
